api/resolvers/search.js

```diff
@@ -192,10 +193,16 @@ export default {
           break
       }
 
-      const { query, quotes, nym, url } = queryParts(q)
+      const { query: _query, quotes, nym, url } = queryParts(q)
+      let query = _query
+
+      const isUrlSearch = url && query.length === 0 // exclusively searching for an url
```

Comment:
Because the all the termQueries are nested within a `must` (logical AND) compound query[^1] we need to adjust the `minimum_should_match` param of the nested queries based on whether we are exclusively searching for a url or not. We use this var to indicate the intent of the user.


[^1]: https://opensearch.org/docs/latest/query-dsl/compound/bool/